Graphics/PDF/Draw.hs

```diff
@@ -186,6 +189,11 @@ instance Monad Draw where
                           unDraw (f a) env
     return x = Draw $ \_env -> return x
 
+#if MIN_VERSION_base(4,11,0)
+instance Fail.MonadFail Draw where
+    fail s = Draw $ \_ -> Fail.fail s
```

Comment:
in my view this is the wrong way to go. the reason the original code was actually fine, was that the list is defined to be infinite, so that should be expressed by using, e.g., http://hackage.haskell.org/package/inflist-0.0.1/docs/src/Data-InfList.html#InfList

i've implemented that here: https://github.com/alpheccar/HPDF/pull/27